{
  "id": "MAL-2024-6154",
  "modified": "2024-10-24T01:01:59Z",
  "published": "2024-06-25T13:43:43Z",
  "aliases": [
    "SNYK-PYTHON-TLSBYPASS-5295969"
  ],
  "summary": "Malicious code in tls-bypass (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "tls-bypass",
        "purl": "pkg:pypi/tls-bypass"
      },
      "versions": [
        "1.0"
      ],
      "database_specific": {
        "source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/tls-bypass/MAL-2024-6154.json"
      }
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://security.snyk.io/vuln/SNYK-PYTHON-TLSBYPASS-5295969"
    },
    {
      "type": "WEB",
      "url": "https://github.com/DataDog/malicious-software-packages-dataset"
    }
  ]
}